stable/kubernetes-dashboard/templates/svc.yaml

```diff
@@ -20,6 +20,7 @@ spec:
   ports:
   - port: {{ .Values.service.externalPort }}
     targetPort: 8443
+    name: "https"
```

Comment:
The only real fix required is this one. The rest can be configured via the values.

Reply:
Please note that previous PR set an "insecure" access using plain HTTP. Naming it https makes no sense.